doc/porting/cmsis_rtos_v2.rst

```diff
@@ -0,0 +1,85 @@
+.. _cmsis_rtos_v2:
+
+CMSIS RTOS v2
+##########################
+
+Cortex-M Software Interface Standard (CMSIS) RTOS is a vendor-independent
+hardware abstraction layer for the ARM Cortex®-M processor series and defines
+generic tool interfaces. Though it was originally defined for ARM Cortex-M
+microcontrollers alone, it could be easily extended to other microcontrollers
+making it generic. For more information on CMSIS RTOS v2, please refer
+http://www.keil.com/pack/doc/CMSIS/RTOS2/html/index.html
+
+Features not supported in Zephyr implementation
+***********************************************
+
+Kernel
+   osKernelGetState, osKernelSuspend, osKernelResume are not supported.
+   Also, osKernelInitialize and osKernelStart are pretty much empty.
```

Comment:
I suspect these two APIs should just be added to the "not supported" list in the previous sentence. What does "pretty much empty" mean?

Reply:
@rgundi: As this change is part of your commits, would you please comment on?

Reply:
sure. will address it in my PR.